contributed/matmul.py

```diff
@@ -0,0 +1,86 @@
+from neuronxcc.nki import baremetal, benchmark
+import neuronxcc.nki.language as nl
+import neuronxcc.nki.isa as ni
+import numpy as np
+
+@benchmark(save_neff_name='file.neff', save_trace_name='profile.ntff', additional_compile_opt=' --disable-internal-io-dge ')
```

Comment:
What was the reasoning behind using `disable-internal-io-dge`?

Reply:
Ah, I see! Could you run the kernel and test the performance with the latest compiler without `disable-internal-io-dge` nor `disable-dge`?

Reply:
Okay. I just removed it for now. Thanks!